### PR TITLE
Updated the mentee approval/rejection emails templates

### LINKED ISSUE #382: Update the emails templates
**Is your feature request related to a problem? Please describe.**
- We need to update some emails templates

**Describe the solution you'd like**
- Update the following email templates

**Mentee Rejection Mail**

> Dear [Applicant Name],
> We wanted to take a moment to thank you for your interest in the ScholarX program and for submitting your application. We appreciate the time and effort you put into it.
> After a careful review of your application and considering all of the candidates, we regret to inform you that we are unable to offer you admission at this time. We received a large number of qualified applicants, and unfortunately, we could only accept a limited number of students.
> However, we want to encourage you not to be discouraged by this decision. We recognize that the admissions process can be competitive, and we understand that this news may be disappointing. Please know that this does not reflect on your abilities, potential, or value as an individual.
> We do offer the possibility for you to apply again next year if you meet the eligibility criteria. We invite you to stay engaged with us by attending our events, reaching out to our admissions team, and taking advantage of any opportunities to connect with our current students and alumni.
> Thank you again for considering our program and for the time you invested in your application. We wish you all the best in your future endeavours.
> Sincerely,
> [Your Name]
> 

**Mentee Acceptance Email**

> Dear [Applicant Name],
> We are delighted to inform you that you have been selected for our undergraduate program, and we extend our heartfelt congratulations to you!
> We received a large number of qualified applicants, and after a thorough review of all candidates, we are thrilled to offer you a place in our program. Your application stood out amongst the others, and we are confident that you will contribute positively to our program.
> We believe that you have great potential to succeed in your academic and professional pursuits, and we are excited to have you join our community of learners and scholars.
> To emphasize the importance of completing the program, you have received a valuable opportunity. If, for any reason, you are uncertain about completing the program within the 6-month timeline, please inform our admissions team as soon as possible, so we can provide the opportunity to another deserving student.
> Once again, congratulations on your selection! We cannot wait to have you on board.
> Best regards,
> [Your Name]

**Additional context**
- Double-check the spacings of the content

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by chirandevinda**
## Purpose

The purpose of this PR is to fix #382

## Goals
Update the emails templates

## Approach

have update the emails templates

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by anjula-sack — CHANGES_REQUESTED** [review]
@chirandevinda can you add the following line to the every email.

> To ensure that you receive our emails and they do not go to your spam folder, please add sustainableedufoundation@gmail.com to your email whitelist.

**Comment by chirandevinda** [comment]
I will

**Comment by chirandevinda** [comment]
> every email

means every email which going to send or just the ones that I edited?

**Comment by anjula-sack** [comment]
> > every email
> 
> means every email which going to send or just the ones that I edited?

means every email which going to send @chirandevinda

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 1]
```diff
@@ -209,12 +211,12 @@ public void sendOnGoingEmails(long id, Optional<Program> program) throws IOExcep
...
+            "However, we want to encourage you not to be discouraged by this decision. We recognize that the admissions process can be competitive, and we understand that this news may be disappointing. Please know that this does not reflect on your abilities, potential, or value as an individual." + "<br /><br />" +
+            "We do offer the possibility for you to apply again next year if you meet the eligibility criteria. We invite you to stay engaged with us by attending our events, reaching out to our admissions team, and taking advantage of any opportunities to connect with our current students and alumni." + "<br /><br />" +
+            "Thank you again for considering our program and for the time you invested in your application. We wish you all the best in your future endeavours." +
+            "To ensure that you receive our emails and they do not go to your spam folder, please add sustainableedufoundation@gmail.com to your email whitelist.";
```
Comment (anjula-sack) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
            " To ensure that you receive our emails and they do not go to your spam folder, please add sustainableedufoundation@gmail.com to your email whitelist.";
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 2]
```diff
@@ -239,7 +241,8 @@ public void sendConfirmationEmails(long profileId, Optional<Program> program) th
                 "You can view/edit your application by visiting the <b>ScholarX dashboard.</b> " +
                 "Reach out to us via " +
                 "<a href=\"mailto:sustainableedufoundation@gmail.com\">sustainableedufoundation@gmail.com</a> " +
-                "for any clarifications.";
+                "for any clarifications." +
```
Comment (anjula-sack) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                "for any clarifications. " +
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 3]
```diff
@@ -180,16 +185,13 @@ public void sendOnGoingEmails(long id, Optional<Program> program) throws IOExcep
...
+            "We are delighted to inform you that you have been selected for our undergraduate program, and we extend our heartfelt congratulations to you!"+ "<br /><br />"+
+            "We received a large number of qualified applicants, and after a thorough review of all candidates, we are thrilled to offer you a place in our program. Your application stood out amongst the others, and we are confident that you will contribute positively to our program." + "<br /><br />"+
+            "We believe that you have great potential to succeed in your academic and professional pursuits, and we are excited to have you join our community of learners and scholars." + "<br /><br />"+
+            "To emphasize the importance of completing the program, you have received a valuable opportunity. If, for any reason, you are uncertain about completing the program within the 6-month timeline, please inform our admissions team as soon as possible, so we can provide the opportunity to another deserving student."+"<br /><br />"+
+            "Once again, congratulations on your selection! We cannot wait to have you on board." +
```
Comment (anjula-sack) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
            "Once again, congratulations on your selection! We cannot wait to have you on board. " +
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 4]
```diff
@@ -162,7 +166,8 @@ public void sendOnGoingEmails(long id, Optional<Program> program) throws IOExcep
             String message = "Dear " + mentor.getProfile().getName() + ",<br /><br />" +
                     "<b>Congratulations!</b><br />Your list of students is now finalised. " +
                     "You can check your mentees and their contact details by visiting the <b>ScholarX dashboard.</b> " +
-                    "Please make the first contact with them as we have instructed them to wait for your email.";
+                    "Please make the first contact with them as we have instructed them to wait for your email." +
```
Comment (anjula-sack) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                    "Please make the first contact with them as we have instructed them to wait for your email. " +
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 5]
```diff
@@ -138,7 +141,8 @@ public void sendMenteeSelectionEmails(long id, Optional<Program> program) throws
                     "as well during the second round of matching.</li><br />"+
                     "We appreciate your enthusiasm in being a part of this journey and kindly request your cooperation in completing this matching process as well.<br />"+
                     "If you have any further queries please don't hesitate to contact us at "+
-                    "<a href=\"mailto:sustainableedufoundation@gmail.com\">sustainableedufoundation@gmail.com</a>";
+                    "<a href=\"mailto:sustainableedufoundation@gmail.com\">sustainableedufoundation@gmail.com</a>" +
+                    "To ensure that you receive our emails and they do not go to your spam folder, please add sustainableedufoundation@gmail.com to your email whitelist.";
```
Comment (anjula-sack) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                    " To ensure that you receive our emails and they do not go to your spam folder, please add sustainableedufoundation@gmail.com to your email whitelist.";
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 6]
```diff
@@ -101,7 +103,8 @@ public void sendMenteeFiltrationEmails(long id, Optional<Program> program) throw
                     "Mentors will soon review your applications and we will keep you posted on the progress via email. " +
                     "Until then, read more about student experience <a href=\"https://medium.com/search?q=scholarx\">here</a> and reach out to us via " +
                     "<a href=\"mailto:sustainableedufoundation@gmail.com\">sustainableedufoundation@gmail.com</a> " +
-                    "for any clarifications.";
+                    "for any clarifications." +
```
Comment (anjula-sack) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                    "for any clarifications. " +
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 7]
```diff
@@ -76,7 +77,8 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
                         "reflect on your abilities, potential or value as an individual. As you progress ahead on your academic or professional journey, we would be glad to have you as a mentor for future ScholarX programs.<br /><br />"+
                         "We appreciate your interest in our program and would like to wish you all the best in your future endeavors. We are grateful for the opportunity "+
                         "to consider you for our program and encourage you to keep pursuing your goals and aspirations.<br /><br />" +
-                        "Thank you again for considering our program and for the time you invested in your application. We hope you find success and fulfillment in your academic and professional pursuits." ;
+                        "Thank you again for considering our program and for the time you invested in your application. We hope you find success and fulfillment in your academic and professional pursuits." +
+                        "To ensure that you receive our emails and they do not go to your spam folder, please add sustainableedufoundation@gmail.com to your email whitelist.";
```
Comment (anjula-sack) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        " To ensure that you receive our emails and they do not go to your spam folder, please add sustainableedufoundation@gmail.com to your email whitelist.";
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 8]
```diff
@@ -54,7 +54,8 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
...
                         "We look forward to seeing the unique perspective and insights you will bring to the mentees and to the program. We believe that you will flourish in this year's edition of ScholarX, and we are thrilled to be a part of your academic or professional journey.<br /><br />" +
-                        "Once again, congratulations on your selection! We cannot wait to have you on board. We will keep you informed on the next steps, and in the meantime would like to invite you to go through some of the resources that would be useful to thrive as a great mentor in  " + program.get().getTitle()  ;
+                        "Once again, congratulations on your selection! We cannot wait to have you on board. We will keep you informed on the next steps, and in the meantime would like to invite you to go through some of the resources that would be useful to thrive as a great mentor in  " + program.get().getTitle() + "<br /><br />" +
+                        "To ensure that you receive our emails and they do not go to your spam folder, please add sustainableedufoundation@gmail.com to your email whitelist.";
```
Comment (anjula-sack) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        " To ensure that you receive our emails and they do not go to your spam folder, please add sustainableedufoundation@gmail.com to your email whitelist.";
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]
I think we need the spaces I suggested @chirandevinda

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 9]
```diff
@@ -54,7 +54,8 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
...
                         "We look forward to seeing the unique perspective and insights you will bring to the mentees and to the program. We believe that you will flourish in this year's edition of ScholarX, and we are thrilled to be a part of your academic or professional journey.<br /><br />" +
-                        "Once again, congratulations on your selection! We cannot wait to have you on board. We will keep you informed on the next steps, and in the meantime would like to invite you to go through some of the resources that would be useful to thrive as a great mentor in  " + program.get().getTitle()  ;
+                        "Once again, congratulations on your selection! We cannot wait to have you on board. We will keep you informed on the next steps, and in the meantime would like to invite you to go through some of the resources that would be useful to thrive as a great mentor in  " + program.get().getTitle() + "<br /><br />" +
```
Comment (Piumal1999) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        "Once again, congratulations on your selection! We cannot wait to have you on board. We will keep you informed on the next steps, and in the meantime would like to invite you to go through some of the resources that would be useful to thrive as a great mentor in  " + program.get().getTitle() + ". <br /><br />" +
```

**Comment by Piumal1999** [comment]
Looks good to me. @anjula-sack shall we get this merged?

**Review by anjula-sack — APPROVED** [review]
Thank you! @chirandevinda

**Pull request merged**